singolo/sass/footer/_footer.scss

```diff
@@ -0,0 +1,71 @@
+.footer{
+    background-color: #2D303A;
+}
+.footer-content{
+    width: 1020px;
+margin: 0px auto;
+    height: 76px;
+    
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0px 40px;
+    box-sizing: border-box;
+    @media (max-width: 1019px) {
+        width: 100vw;
+      }
+      @media (max-width: 767px) {
+        justify-content: center;
+      }
+}
+
+.footer-content__description{
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: 900;
+    font-size: 13px;
+    line-height: 18px;
+    color: #666D89;
+    @media (max-width: 767px) {
+        display: none;
+      }
+
+
+}
+.footer-content__social-buttons{
+    width: 152px;
+    height: 32px;
+    display: flex;
+    justify-content: space-between;
+}
+.button{
```

Comment:
In the _buttons.scss_ you have the same selector.

Reply:
fixed